.gitignore

```diff
@@ -1,3 +1,6 @@
+#prisma migrations
+prisma/migrations/[0-9]*_[a-zA-Z]*/
```

Comment:
sirh wait what the, why are you gitignoring migrations?

Reply:
Oh no!

Reply:
> > sirh wait what the, why are you gitignoring migrations?
> 
> mmlol

i wasn't joking 🙃